homeassistant/components/go2rtc/__init__.py

```diff
@@ -99,10 +111,26 @@ async def on_stop(event: Event) -> None:
 
         url = "http://localhost:1984/"
 
+    hass.data[_DATA_GO2RTC] = url
```

Comment:
Can we use the config entry data instead?

Reply:
I don't agree, there's a single config entry and there's no reason to persistently store the URL.